index.html

```diff
@@ -879,6 +878,20 @@ <h2 id="constrainable-properties">Constrainable Properties for Captured
             {{CursorCaptureConstraint}} it is capable of supporting for this
             [=display surface=].</td>
           </tr>
+
+          <tr id="def-constraint-pixel-ratio">
+            <td><dfn class="export">screenPixelRatio</dfn>
+            </td>
+
+            <td>{{double}}</td>
+
+            <td>As a setting, it is the result of dividing the size of a 
+            <a data-xref-type="css-value" data-xref-for="<length>" data-lt="px">CSS pixel</a>
+            at a <a="https://drafts.csswg.org/cssom-view-1/#page-zoom">page zoom</a>
+            of 1.0 by the vertical size of a pixel from the [=display surface=].
```

Comment:
I would use the same terminology as [CSS](https://drafts.csswg.org/cssom-view/#dom-window-devicepixelratio):
`the size of a [CSS pixel](https://drafts.csswg.org/css-values-4/#px) at the current [page zoom](https://drafts.csswg.org/cssom-view/#page-zoom) and using a [scale factor](https://drafts.csswg.org/cssom-view/#scale-factor) of 1.0.`

scale-factor might handle pinch zoom.

Reply:
But are you OK with current page zoom or do you prefer page zoom 1.0?

Reply:
For this `pixelRatio` definition, I would go with 1.0 for both values.